srcpkgs/SDL2_mixer/template

```diff
@@ -1,30 +1,20 @@
 # Template file for 'SDL2_mixer'
 pkgname=SDL2_mixer
-version=2.0.4
-revision=4
+version=2.6.2
+revision=1
 build_style=gnu-configure
+configure_args="--disable-static --prefix=/usr"
```

Comment:
It's failing because you disabled building a static library, but are trying to move it to a subpackage in SDL2_mixer-devel's pkg_install. Why are you disabling this?

```
mv: cannot stat '/destdir//SDL2_mixer-2.6.2/usr/lib/*.a': No such file or directory
```